lib/nodetypes/vespa_node.rb

```diff
@@ -217,6 +217,11 @@ def log_matches(regexp)
     logcontent.scan(regexp).size
   end
 
+  def trigger_flush
```

Comment:
I am not sure we want a _trigger_flush_ in this class, as it only works when its actually a SearchNode on the same host. nodetypes/searchnode.rb already has this function defined.